src/axom/primal/operators/detail/winding_number_impl.hpp

```diff
@@ -42,7 +42,7 @@ namespace detail
  * is the signed angle subtended by the query point to each endpoint.
  * Colinear points return 0 for their winding number.
  *
- * \return double The winding number
+ * \return double The GWN
```

Comment:
Minor: My initial read of this doxyline was that the return value is twice (double) that of the GWN
(not sure we should do anything about it, maybe remove the return type, i.e. the word `double`)

Reply:
Hah.  I'd be fine with removing "double".

Reply:
This caught me off guard a couple of times too. Removing `double` it is!